src/linker/Linker/LoggingReflectionPatternRecorder.cs

```diff
@@ -45,7 +45,7 @@ public void RecognizedReflectionAccessPattern (IMemberDefinition source, Instruc
 		public void UnrecognizedReflectionAccessPattern (IMemberDefinition source, Instruction sourceInstruction, IMetadataTokenProvider accessedItem, string message)
 		{
 			var origin = new MessageOrigin (source, sourceInstruction?.Offset);
-			_context.LogWarning (message, 2006, origin, "Unrecognized reflection pattern");
+			_context.LogWarning (message, 2006, origin, MessageSubCategory.UnrecognizedReflectionPattern);
```

Comment:
I think this should be TrimCorrectness subcategory. Unrecognized reflection is a problem for trim correctness, but not other types of analysis.